Doc/library/gettext.rst

```diff
@@ -131,7 +131,7 @@ install themselves in the built-in namespace as the function :func:`!_`.
 
    If *localedir* is not given, then the default system locale directory is used.
    [#]_  If *languages* is not given, then the environment variable :envvar:`LANGUAGE`
-   is searched, it falls back to :func:`locale.getlocale`, which in turn falls
+   is searched, it falls back to :func:`locale.setlocale`, which in turn falls
```

Comment:
Mentioning `locale.setlocale()` here looks weird, as we do not change the current locale. It is just an implementation detail. Maybe just say "the current locale"?

"... it falls back to the current locale or to the environment variables ..."?